chillTime.py

```diff
@@ -146,88 +137,14 @@ def get_aggregate(self, obs_type, timespan, aggregate_type, db_manager, **option
             'stop': timespan.stop
         }
 
-        sql_stmt = ChillTime.sql_stmts[dbtype].format(**interp_dict)
-
-        try:
-            row_gen = db_manager.genBatchRecords(timespan.start, timespan.stop)
-        except:
-            log.error("weedb No Column Error rasied")
-
-###*** Need the correct error hanlding for genBatchRecords here. No clue what it should be   
-        """
-        except weedb.NoColumnError:
-            log.error("weedb No Column Error rasied")
-            raise weewx.UnknownType(aggregate_type)
+        chill_total = 0
 
-        if not row or None in row:
-            value = None"""
+        for record in db_manager.getBatchRecords(*timespan):
+            chill_delta = self.get_scalar('chillTime', record, db_manager)
```

Comment:
This returns a tuple, cannot be added to
```suggestion
            chill_delta_vt = self.get_scalar('chillTime', record, db_manager)
            chill_delta = chill_delta_vt[0]
```